sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/fileSourceInterfaces.scala

```diff
@@ -230,6 +229,15 @@ trait FileFormat {
   }
 
   /**
+   * Return a `SerializablePathFilter` class to filter qualified files for this format.
+   */
+  def getPathFilter(options: Map[String, String]): SerializablePathFilter = {
+    new SerializablePathFilter {
+      override def accept(path: Path): Boolean = true
+    }
```

Comment:
We can replace this with `passThroughPathFilter` after moving it into this file.